easybuild/easyconfigs/o/OpenMM/OpenMM-8.0.0-foss-2022a.eb

```diff
@@ -0,0 +1,69 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# Update to 7.5.1
+# J. Sassmannshausen / GSTT
+
+easyblock = 'CMakeMake'
+
+name = 'OpenMM'
+version = '8.0.0'
+
```

Comment:
This needs a `versionsuffix` for `CUDA` (and the same for the filename).

Reply:
done